glean-core/src/metrics/event.rs

```diff
@@ -84,6 +84,14 @@ impl EventMetric {
                 }
             }
         });
+
+        let id = self.meta().base_identifier();
+        crate::launch_with_glean(move |_| {
+            let event_listeners = crate::event_listeners().lock().unwrap();
```

Comment:
Guess this means any run with events in it will init that empty `OnceCell<Mutex<HashMap<String, Listener>>>` eventually. Probably not worth it to special case a "Don't init this if it isn't init` path, though, huh.

Reply:
We could, I don't see much overhead for this at this point but happy to keep this optimization in my back pocket, should it turn out to be useful.